src/seml/experiment.py

```diff
@@ -31,6 +37,78 @@ class LoggerOptions(Enum):
     RICH = 'rich'
 
 
+def process_id():
```

Comment:
Is there a rationale to fragment this functionality into many functions like this?

Reply:
Convenience so the user doesn't have to figure out the logic with env vars etc.